.vscode/settings.json

```diff
@@ -7,5 +7,12 @@
         },
         "editor.defaultFormatter": "charliermarsh.ruff",
     },
-    "flake8.args": ["--max-line-length=120"]
-}
\ No newline at end of file
+    "flake8.args": [
+      "--max-line-length=120"
+    ],
+    "python.testing.pytestArgs": [
+      "tests"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true
```

Comment:
Just curious, is this diff intentional?

Reply:
Yea, this hooks up pytest to the testing pane in vscode/positron